src/app/lib/config/services/igbo.js

```diff
@@ -33,32 +33,32 @@ const igbo = {
     error: {
       404: {
         statusCode: '404',
-        title: 'Page cannot be found',
+        title: 'Ahụghị ibe akwụkwọ a',
         message:
-          "Sorry, we're unable to bring you the page you're looking for. Please try:",
+          'E wela iwe, anyị enweghị ike iwetara gị ibe akwụkwọ ị na-achọ. Nwalie ọzọ:',
         solutions: [
-          'Double checking the url',
-          'Hitting the refresh button in your browser',
-          'Searching for this page using the BBC search bar',
+          'Tulee akara URL ahụ ọzọ',
+          "Kpatụ ọdụmbị 'Refresh' dị na braụza gị aka",
+          'Jiri BBC "Search Bar" chọọ ibe akwụkwọ',
         ],
-        callToActionFirst: 'Alternatively, please visit the ',
-        callToActionLinkText: 'BBC News homepage.',
-        callToActionLast: '',
-        callToActionLinkUrl: 'https://www.bbc.com/news',
+        callToActionFirst: "Maọbụ, biko gaa n'obi ",
+        callToActionLinkText: 'BBC News Ìgbò',
+        callToActionLast: '.',
+        callToActionLinkUrl: 'https://www.bbc.com/igbo',
       },
       500: {
         statusCode: '500',
-        title: 'Internal server error',
+        title: 'Mperi',
         message:
-          "Sorry, we're currently unable to bring you the page you're looking for. Please try:",
+          'E wela iwe, anyị enweghị ike iwetara gị ibe akwụkwọ ị na-achọ.Gbalịa ọzọ:',
```

Comment:
```suggestion
          'E wela iwe, anyị enweghị ike iwetara gị ibe akwụkwọ ị na-achọ. Gbalịa ọzọ:',
```

pretty sure this is a typo in the translations spreadsheet cc @clareevansbbc

Reply:
Also pretty sure that it's a typo. Went ahead and committed it as it's just adding a space